### PR TITLE
Updated Gonzalo about text

### PR DESCRIPTION
**Pull request by gonzalo1330**

### CODE REVIEW

File: src/pages/About.jsx [hunk 1]
```diff
@@ -20,7 +20,9 @@ const developers = [
   {
     name: 'Gonzalo Lara',
     img: testPic,
-    description: 'Todo: about Gonzalo Lara',
+    description: 'My passion for computers has lead me to pursue a career in Software Engineering. I am currently '
+                 'a UWB CSS student completing my final year in the Computer Science and Software Engineering program. I am 24 years old, '
+                 'bilingual and currently a part of the DACA program. I intend to contribute as much as possible to building a better tomorrow.',
```
Comment (josueetcom) [inline src/pages/About.jsx:25]: Try using a [template literal](https://flaviocopes.com/javascript-template-literals/#multiline-strings)

Comment (gonzalo1330) [reply src/pages/About.jsx:25]: Ok I did that in the other pull request, this one can get deleted.

Comment (josueetcom) [reply src/pages/About.jsx:25]: I suggest leaving that change in this pull request. It adds extra noise to the other PR and merging will depend on when the other changes get approved.

Comment (gonzalo1330) [reply src/pages/About.jsx]: Got it, Well I included the changes to this pull if you want to take a look and merge or request further revisions.

File: src/pages/About.jsx [hunk 2]
```diff
@@ -20,7 +20,11 @@ const developers = [
   {
     name: 'Gonzalo Lara',
     img: testPic,
-    description: 'Todo: about Gonzalo Lara',
+    description: 'My passion for computers has lead me to pursue a career in Software '
+      + 'Engineering. I am currently a UWB CSS student completing my final year in the '
+      + 'Computer Science and Software Engineering program. I am 24 years old, bilingual '
+      + 'and currently a part of the DACA program.  I intend to contribute as much as '
+      + 'possible to building a better tomorrow.',
```
Comment (josueetcom) [inline src/pages/About.jsx:27]: This'll be tricky to edit later on. I recommend keeping this in a single line. If the linter complains we can disable it for these lines

Comment (josueetcom) [reply src/pages/About.jsx:27]: Surround the `developers` array with these lines:
```js
/* eslint-disable max-len */
const developers = [
  ...
];
/* eslint-enable max-len */
```

That'll allow the description strings to go beyond the line length.

Comment (gonzalo1330) [reply src/pages/About.jsx]: Ok I include it as a single line and im awaiting the lint test results if I do get an error due to that line Ill update that in the comment. Also change the surround of `developers` array with the lines you provided.

Comment (gonzalo1330) [reply src/pages/About.jsx]: Update, the lint error does not complain due to the line exceeding the line length and all three test passed.

**Review by josueetcom — APPROVED** [review]

**Pull request merged**